streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -47,7 +50,7 @@
  *     final KafkaStreams streams = ...;
  *     streams.start()
  *     ...
- *     final String queryableStoreName = table.getStoreName(); // returns null if KTable is not queryable
+ *     final String queryableStoreName = table.queryableStoreName(); // returns null if KTable is not queryable
```

Comment:
Ok looked into it. I'll have to maintain and propagate a boolean throughout a lot of code to signal the original intention. In addition, there are internal calls that really want to get the store name (whether the user specified a queryable name or not). Are we sure it's worth it? We had a 50-50% split when it came to the question of "can internal stores still be queryable if user really wants to dig the internal name?" cc @mjsax @dguy